modules/client/examples/01-client/10-prepare-installation.ts

```diff
@@ -1,34 +1,40 @@
 /* MARKDOWN
 ---
-title: PrepareInstallation
+title: Prepare Installation
 ---
 ### Prepare installation of a plugin
+
+The `prepareInstallation` method performs the prior steps so that a DAO proposal can eventually apply the installation of a Plugin to the given DAO.
+A governance proposal will need to be created with an action calling `applyInstallation`.
```

Comment:
Where can `applyInstallation` be found? How would devs go about using the output of the `prepareInstallation` function to install the plugin in their DAO?